assemblies/assembly-add-custom-app-file-openshift.adoc

```diff
@@ -0,0 +1,25 @@
+[id='assembly-add-custom-app-file-openshift_{context}']
+= Adding a custom application configuration file to {ocp-brand-name}
+
+To access the {product}, you must add a custom application configuration file to {ocp-brand-name}. In {ocp-short}, you can use the following content as a base template to create a ConfigMap named `app-config-rhdh`:
+
+[source]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: app-config-rhdh
+data:
+  app-config-rhdh.yaml: |
+    app:
+      title: {product}
+----
+
+You can add the custom application configuration file to {ocp-short} in one of the following ways:
+
+* The {product} Operator
+* The {product} Helm chart.
```

Comment:
I'd update these to "By using The {product} Operator." instead -  it just reads better IMHO